src/js/components/FocusedContainer.js

```diff
@@ -16,11 +16,19 @@ const isFocusable = (element) => {
     case 'SELECT':
     case 'TEXTAREA':
       return true;
+    case 'DIV':
```

Comment:
should this be any element with tabIndex >= 0? just to be safe if someone has programmatically added it

Reply:
yes thats on the next line